src/sass/modules/_m-schemaform.scss

```diff
@@ -231,3 +231,12 @@ label {
     padding-left: 25px !important;
   }
 }
+
+.usa-button.schemaform-file-remove-button {
+  width: auto;
```

Comment:
Yeah, the UI is not good, I was just going for "doesn't look too broken." Alex and Stacy will make it look nice once they're done researching/testing.